### DISCUSSION TITLE
Indent Guides Highlight only Active Block

### DISCUSSION THREAD

**Discussion by PhysicallyActive** [LazyVim/LazyVim#5168] [Q&A] [answered]
Hi,

I have a question related to Snacks.indent.
The documentation clearly states that the guides could be toggled using `<leader>ug`. However, I wonder if it's also possible to configure Snacks.indent to only provide guides for the current block, i.e., similar to how "chunk" adds an arrow to highlight the current block? I only want the "chunk" arrow, and none of the other guides, if that's possible?

E.g., This is how my setup currently looks:
![image](https://github.com/user-attachments/assets/1537f1d7-9b23-4e1d-9ede-8f9f6a712aa1)
I'd like to remove all guides except the line highlighting the chunk.

Thank you for a great eco-system and let me know if I can clarify something about my request (it's likely that I've missed some configuration option).

**Comment by folke** [comment, marked as answer]
`opts.indent.indent.enabled = false`

**Reply by PhysicallyActive** [reply]
Thanks a lot, just what I was looking for!
![image](https://github.com/user-attachments/assets/bd7063c1-cc8a-450c-8293-5826a8ac0c83)
